### PR TITLE
friendly editorial suggestions

### PR DESCRIPTION
**Pull request by seanturner**
I did a pass through the document to s7. All of these are intended to be editorial even though some of them affect the presentation syntax. Hopefully, I didn't introduce any errors.

### CODE REVIEW

**Review by tgeoghegan — APPROVED** [review]

File: draft-ietf-ppm-dap.md [hunk 1]
```diff
@@ -1549,7 +1550,7 @@ Collect sub-protocol messages must be confidential and mutually authenticated.
 
 HTTPS provides confidentiality and authenticates the leader to the collector.
 Additionally, the leader encrypts its aggregate share to a public key held by
-the collector.
+the collector using [HPKE].
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:1553]:
```suggestion
the collector using {{!HPKE}}.
```

File: draft-ietf-ppm-dap.md [hunk 2]
```diff
@@ -1563,7 +1564,7 @@ detailed in {{collect-aggregate}}. The aggregate share must be confidential from
 everyone but the helper and the collector.
 
 Confidentiality is achieved by having the helper encrypt its aggregate share to
-a public key held by the collector.
+a public key held by the collector using [HPKE].
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:1567]:
```suggestion
a public key held by the collector using {{!HPKE}}.
```

File: draft-ietf-ppm-dap.md [hunk 3]
```diff
@@ -1984,7 +1985,7 @@ output parts and so could request aggregations over few inputs.
 1. Known input injection. Collectors may collude with clients to send known
    input to the aggregators, allowing collectors to shrink the effective
    anonymity set by subtracting the known inputs from the final output. Sybil
-   attacks [Dou02] could be used to amplify this capability.
+   attacks {{Dou02}} could be used to amplify this capability.
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:1988]:
```suggestion
   attacks [Dou02] could be used to amplify this capability.
```

File: draft-ietf-ppm-dap.md [hunk 4]
```diff
@@ -822,12 +822,12 @@ to authenticate the aggregator they upload to.
...
-share to a public key held by the respective aggregator. Clients fetch the
-public keys from each aggregator over HTTPS, allowing them to authenticate the
-server.
+share to a public key held by the respective aggregator using [HPKE].
+Clients fetch the public keys from each aggregator over HTTPS, allowing them to
+authenticate the server.
 
 Aggregators MAY require clients to authenticate when uploading reports. This is
-an effective mitigation against Sybil attacks in deployments where it is
+an effective mitigation against Sybil {{Dou2}} attacks in deployments where it is
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:830]:
```suggestion
an effective mitigation against Sybil [Dou2] attacks in deployments where it is
```

File: draft-ietf-ppm-dap.md [hunk 5]
```diff
@@ -822,12 +822,12 @@ to authenticate the aggregator they upload to.
 HTTPS provides confidentiality between the DAP client and the leader, but this
 is not sufficient since the helper's report shares are relayed through the
 leader. Confidentiality of report shares is achieved by encrypting each report
-share to a public key held by the respective aggregator. Clients fetch the
-public keys from each aggregator over HTTPS, allowing them to authenticate the
-server.
+share to a public key held by the respective aggregator using [HPKE].
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:825]:
```suggestion
share to a public key held by the respective aggregator using {{!HPKE}}.
```

File: draft-ietf-ppm-dap.md [hunk 6]
```diff
@@ -548,13 +548,13 @@ subsection below.
...
 a "batch interval" that determines the time range for reports included in the
-batch: For each report in the batch, the time at which that report was generated
+batch. For each report in the batch, the time at which that report was generated
 (see {{upload-flow}}) must fall within the batch interval specified by the
 Collector.
 
 Typically the Collector issues queries for which the batch intervals are
 continuous, monotonically increasing, and have the same duration. For example,
-the sequence of batch intervals. For example, the sequence of batch intervals
+the sequence of batch intervals
```
Comment (cjpatton) [inline draft-ietf-ppm-dap.md:557]: Hmm, why make this change? I'm not a huge fan :)

File: draft-ietf-ppm-dap.md (hunk 4 above)
Comment (cjpatton) [reply draft-ietf-ppm-dap.md:830]: Don't we want {{Dou2}} so that the existence of the reference is enforced?

**Review by cjpatton — COMMENTED** [review]
GREAT feedback, thanks for this. Just a few nits. (Also note that the build is failing.)

**Comment by seanturner** [comment]
> (Also note that the build is failing.)

The Duo2 reference was [] originally. I changed it to {{}}, and then got the build error. I think that's because I didn't include the ?. I change it and see if it addresses the build error.

**Comment by seanturner** [comment]
Yep adding the ? with the ref fixed the build error.

**Comment by cjpatton** [comment]
Thanks @seanturner! Please rebase (there are merge conflicts) and squash when you have a second.

File: draft-ietf-ppm-dap.md [hunk 7]
```diff
@@ -827,12 +827,12 @@ to authenticate the aggregator they upload to.
...
-share to a public key held by the respective aggregator. Clients fetch the
-public keys from each aggregator over HTTPS, allowing them to authenticate the
-server.
+share to a public key held by the respective aggregator using {{!HPKE}}.
+Clients fetch the public keys from each aggregator over HTTPS, allowing them to
+authenticate the server.
 
 Aggregators MAY require clients to authenticate when uploading reports. This is
-an effective mitigation against Sybil attacks in deployments where it is
+an effective mitigation against Sybil {{?Dou2}} attacks in deployments where it is
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:835]:
```suggestion
an effective mitigation against Sybil {{?Dou02}} attacks in deployments where it is
```

File: draft-ietf-ppm-dap.md [hunk 8]
```diff
@@ -827,12 +827,12 @@ to authenticate the aggregator they upload to.
...
-share to a public key held by the respective aggregator. Clients fetch the
-public keys from each aggregator over HTTPS, allowing them to authenticate the
-server.
+share to a public key held by the respective aggregator using {{!HPKE}}.
+Clients fetch the public keys from each aggregator over HTTPS, allowing them to
+authenticate the server.
 
 Aggregators MAY require clients to authenticate when uploading reports. This is
-an effective mitigation against Sybil attacks in deployments where it is
+an effective mitigation against Sybil {{?Dou02}} attacks in deployments where it is
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:835]:
```suggestion
an effective mitigation against Sybil {{Dou02}} attacks in deployments where it is
```

File: draft-ietf-ppm-dap.md [hunk 9]
```diff
@@ -1991,7 +1992,7 @@ output parts and so could request aggregations over few inputs.
 1. Known input injection. Collectors may collude with clients to send known
    input to the aggregators, allowing collectors to shrink the effective
    anonymity set by subtracting the known inputs from the final output. Sybil
-   attacks [Dou02] could be used to amplify this capability.
+   attacks {{?Dou02}} could be used to amplify this capability.
```
Comment (seanturner) [inline draft-ietf-ppm-dap.md:1995]:
```suggestion
   attacks {{Dou02}} could be used to amplify this capability.
```

**Comment by wangshan** [comment]
@seanturner @cjpatton is `//` an invalid comment for RFC8446 representation language?

**Review by cjpatton — APPROVED** [review]

**Comment by cjpatton** [comment]
@seanturner if you don't want to have to muck around with rebasing, I've opened #339 as an alternative.

**Comment by cjpatton** [comment]
Closed in favor of #339.

**Pull request closed without merging**